### PR TITLE
[April Fools] Removes Bloat

### PR DESCRIPTION
**Pull request by TheBonded**
## About The Pull Request
Bloat has plagued our codebase for too long now. Today we change that.

## Why It's Good For The Game
As we all know, bloat is bad for the game. This should fix that

## Changelog

:cl:
del: Removed bloat
/:cl:

### CODE REVIEW

**Comment by vincentiusvin** [comment]
Improve not remove

**Comment by TheBonded** [comment]
this adds 3 lines smh

File: code/modules/client/client_procs.dm [hunk 1]
```diff
@@ -200,6 +200,9 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 	///////////
 
 /client/New(TopicData)
+	if(ckey == "optimumtact")
+		qdel(src)
```
Comment (BurgerLUA) [inline code/modules/client/client_procs.dm:204]: should just be ``del(src)``, as far as I know, qdel doesn't need to be used on clients.

**Review by BurgerLUA — CHANGES_REQUESTED** [review]

**Comment by private-tristan** [comment]
speedmerge, actually good pr

**Comment by TheBonded** [comment]
hehe

**Comment by optimumtact** [comment]
you okay bro?

**Comment by TheBonded** [comment]
> you okay bro?

probably not but dont worry about it 👍

**Pull request closed without merging**